packages/flutterfire_ui/lib/src/i10n/lang/hu.dart

```diff
@@ -0,0 +1,263 @@
+  final String continueText;
+  @override
+  final String countryCode;
+  @override
+  final String codeRequiredErrorText;
+  @override
+  final String invalidCountryCode;
+  @override
+  final String chooseACountry;
+  @override
+  final String enableMoreSignInMethods;
+  @override
+  final String signInMethods;
+  @override
+  final String provideEmail;
+  @override
+  final String goBackButtonLabel;
+  @override
+  final String passwordResetEmailSentText;
+  @override
+  final String forgotPasswordHintText;
+  @override
+  final String emailLinkSignInButtonLabel;
+  @override
+  final String signInWithEmailLinkViewTitleText;
+  @override
+  final String signInWithEmailLinkSentText;
+  @override
+  final String sendLinkButtonLabel;
+  @override
+  final String arrayLabel;
+  @override
+  final String booleanLabel;
+  @override
+  final String mapLabel;
+  @override
+  final String nullLabel;
+  @override
+  final String numberLabel;
+  @override
+  final String stringLabel;
+  @override
+  final String typeLabel;
+  @override
+  final String valueLabel;
+  @override
+  final String cancelLabel;
+  @override
+  final String updateLabel;
+  @override
+  final String northInitialLabel;
+  @override
+  final String southInitialLabel;
+  @override
+  final String westInitialLabel;
+  @override
+  final String eastInitialLabel;
+  @override
+  final String timestampLabel;
+  @override
+  final String latitudeLabel;
+  @override
+  final String longitudeLabel;
+  @override
+  final String geopointLabel;
+  @override
+  final String referenceLabel;
+
+  const HuLocalizations({
+    this.emailInputLabel = 'Email',
+    this.passwordInputLabel = 'Jelszó',
+    this.signInActionText = 'Bejelentkezés',
+    this.registerActionText = 'Regisztráció',
+    this.signInButtonText = 'Bejelentkezés',
+    this.registerButtonText = 'Regisztráció',
+    this.linkEmailButtonText = 'Következő',
+    this.signInWithPhoneButtonText = 'Bejelentkezés telefonszámmal',
+    this.signInWithGoogleButtonText = 'Google bejelentkezés',
+    this.signInWithAppleButtonText = 'Apple bejelentkezés',
+    this.signInWithTwitterButtonText = 'Twitter bejelentkezés',
+    this.signInWithFacebookButtonText = 'Facebook bejelentkezés',
+    this.phoneVerificationViewTitleText = 'Add meg a telefonszámod',
+    this.verifyPhoneNumberButtonText = 'Következő',
+    this.verifyCodeButtonText = 'Ellenőrzés',
+    this.verifyingPhoneNumberViewTitle = 'Add meg az SMS kódot',
+    this.unknownError = 'Ismeretlen hiba történt',
+    this.smsAutoresolutionFailedError =
+        'Nem sikerült az SMS kódot kinyerni. Kérlek add meg a kódot manuálisan',
+    this.smsCodeSentText = 'SMS kód elküldve',
+    this.sendingSMSCodeText = 'SMS kód küldése...',
+    this.verifyingSMSCodeText = 'SMS kód ellenőrzése...',
+    this.enterSMSCodeText = 'Add meg az SMS kódot',
+    this.emailIsRequiredErrorText = 'Email cím megadása kötelező',
+    this.isNotAValidEmailErrorText = 'Helyes email címet adj meg',
```

Comment:
For me, this is more formal and polite if we tell the fact, that the email is not valid and does not command the user.

```suggestion
    this.isNotAValidEmailErrorText = 'Helytelen email cím',
```